core/src/main/java/com/netflix/iceberg/TableOperations.java

```diff
@@ -66,10 +66,7 @@
    * @return a {@link com.netflix.iceberg.encryption.EncryptionManager} to encrypt and decrypt
    * data files.
    */
-  default EncryptionManager encryption() {
-    // TODO coming soon
-    throw new UnsupportedOperationException("Encryption is a work in progress.");
-  }
+  EncryptionManager encryption();
```

Comment:
Because this is part of the service-provider interface and not the public API, I think we should consider keeping the default implementation and update it to return the `PlaintextEncryptionManager`. That would cut down on the files that this touches a bit.